### ISSUE TITLE
training loss not decrease 

### ISSUE THREAD

**Issue by wl082013** [yulunzhang/RCAN#76]
Hi I  tried to run the code, it somehow worked after I changed the torch version and some codes. However, during the training, the loss is not descreasing, but increasing (see the attached figure). I dont know  what is the problem, maybe there is something with the code ? 
Really appreiciate your help.

![1565701586(1)](https://user-images.githubusercontent.com/16892862/62946237-4a36a380-be1b-11e9-895f-af4e056572ce.jpg)

**Comment by yhu9** [comment]
I experience a similar issue, and add that the training loss is extremely high. Does model directly predict the uint8 RGB values? If so, why was this found necessary instead of output being within range of [0,1].

**Comment by LcenArthas** [comment]
I had the same problem, and at a certain point in the training, loss became very large.Have you solved the problem yet?

**Comment by Nkriskeeic** [comment]
In my case, the initializer was the problem.
I used LeCunNormal initializer and the loss became extremely large, but KaimingUniform initializer, the loss converged properly.

**Comment by LcenArthas** [comment]
Thank you very much. Could you please tell me where I should make the change?I actually use the pre-trained model to initialize it, but the effect is very bad.

**Comment by BEAUTIFULSB** [comment]
> In my case, the initializer was the problem. I used LeCunNormal initializer and the loss became extremely large, but KaimingUniform initializer, the loss converged properly.

Could you tell me which part of the code is the initialization of the model，please?
